engine/src/main/java/org/hibernate/validator/internal/engine/ValueContext.java

```diff
@@ -171,20 +171,19 @@ public final void markCurrentPropertyAsIterableAndSetIndex(Integer index) {
 	 * Sets the container element information.
 	 *
 	 * @param containerClass the class of the container
-	 * @param typeParameter the actual type parameter (i.e. not one of our internal type parameters)
+	 * @param typeParameterIndex the index of the actual type parameter (i.e. not one of our internal type parameters)
```

Comment:
Seems the (...) part is superfluous now?

Reply:
yes it seems so, as we just pass an already calculated index to this method. Removed the (...) part.